item.py

```diff
@@ -11,7 +12,7 @@ class ItemPassive:
 
     name: str = ""
     unique: bool = False
-    stats: dict = None
+    stats: Stats = None
```

Comment:
I'm not sure it is useful to put stats into passive. A lot of passives (and unique ones) don't really have anything to do with stats